README.md

```diff
@@ -1,26 +1,70 @@
-# apollo-link-rest
+---
+title: REST Link
+---
 
-## Problem to solve
-You have an existing REST set of services that you have worked on for a long time. However the limitations of it are piling up and you think GraphQL is a great solution to make your application faster and easier to develop. With `apollo-link-rest`, you can start trying out GraphQL without a full server. You can use it to prototype and even use it for third party services that don't yet have a GraphQL endpoint.
+## ⚠️ This library is under active development ⚠️
 
-Apollo Link Rest lets you query traditional REST API endpoints while writing GraphQL. It is designed to work with Apollo Client, but can even be used on its own with Apollo Link.
+This library is under active development. For information on progress check out
+[this issue](https://github.com/apollographql/apollo-link-rest/issues/3) or the
+design [here](./designs/initial.md). We would love your help with writing, docs,
+testing, anything! We would love for you, yes you, to be a part of the Apollo
+community!
 
-## Status
-This library is under active development. For information on progress check out [this issue](https://github.com/apollographql/apollo-link-rest/issues/3) or the design [here](./designs/initial.md). We would love your help! If you want to get involved create or comment on an issue with interest! This could be writing, docs, testing, anything! We would love for you, yes you, to be a part of the Apollo community!
+## Purpose
 
-## Contributing
-This project uses TypeScript to bring static types to JavaScript and uses Jest for testing. To get started, clone the repo and run the following commands:
+An Apollo Link to easily try out GraphQL without a full server. It can be used
+to prototype, with third-party services that don't have a GraphQL endpoint or
+in a transition from REST to GraphQL.
 
-```bash
-npm install # you can also run `yarn`
+## Installation
+
+***Not yet published***
```

Comment:
I think we can add add the provisional installation instructions, right below the `Not yet published` disclaimer. That way we're ready for when we launch!
```bash
npm install apollo-link-rest --save # you can also run `yarn add apollo-link-rest`
```